src/source-build-reference-packages/Directory.Build.props

```
@@ -16,6 +16,7 @@
     <CustomizationsSourceFile>$(CustomizationsPrefix).cs</CustomizationsSourceFile>
 
     <ArtifactsReferenceOnlyPackagesDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsPackagesDir)','ReferenceOnly'))</ArtifactsReferenceOnlyPackagesDir>
+    <ManualNuspecTargets>$([MSBuild]::NormalizePath('$(RepositoryEngineeringDir)','ManualNuspec.targets'))</ManualNuspecTargets>
```

Super tiny nit. You don't need a function here as `RepositoryEngineeringDir` is already correctly normalized (It uses the NormalizeDirectory function in the arcade SDK).

```suggestion
    <ManualNuspecTargets>$(RepositoryEngineeringDir)ManualNuspec.targets</ManualNuspecTargets>
```

---

Absolutely.  Silly oversight on my part.